### ISSUE TITLE
log show?

### ISSUE THREAD

**Issue by a897456** [sp-uhh/sgmse#56]
![图片](https://github.com/user-attachments/assets/1386ca83-8d29-484f-9d57-98acf2c1efb6)

**Comment by Yangliusha** [comment]
Hi, May I know whether you can train your model to get similar results as shown in the paper? Do you use the default settings as given in the codes?

**Comment by a897456** [comment]
> Hi, May I know whether you can train your model to get similar results as shown in the paper? Do you use the default settings as given in the codes?

Yes, the model has not been changed in any way. I trained it twice. As can be seen from the two PESQ lines, the only difference between them is the dataset. However, I did not achieve the metrics in the paper, and the PESQ could only reach 3 and then stalled.

**Comment by libai2345** [comment]
What dataset you're using？

**Comment by a897456** [comment]
> What dataset you're using？

Voicebank+DEMAND

**Comment by libai2345** [comment]
Why is my training so much worse than yours?

**Comment by dorispei** [comment]
> Why is my training so much worse than yours?

![图片](https://github.com/user-attachments/assets/fd89e76c-61b4-4137-bf6c-f716cbe15f0d)
I have the same problem. Have you resolved it? Below is the log of my training. The dataset used is VoiceBank-Demand. The purple curve uses the DCUNet backbone, while the orange curve uses NCSN++.

**Comment by a897456** [comment]
> > Why is my training so much worse than yours?
> 
> ![图片](https://private-user-images.githubusercontent.com/44772675/391777561-fd89e76c-61b4-4137-bf6c-f716cbe15f0d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyMTM0OTIsIm5iZiI6MTczMzIxMzE5MiwicGF0aCI6Ii80NDc3MjY3NS8zOTE3Nzc1NjEtZmQ4OWU3NmMtNjFiNC00MTM3LWJmNmMtZjcxNmNiZTE1ZjBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAzVDA4MDYzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjMmU4YmNlZTZjMDBiYmJjNjg5NTIxOWRkYWU5MTZkZTA2MGYzNDkwNjMzN2YzZTU5ZjEzOGRiOTY3MGU1NGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FglgQsMPEMWV4a4pSnKm2bivxhbdia7GPFWuwFHTloI) I have the same problem. Have you resolved it? Below is the log of my training. The dataset used is VoiceBank-Demand. The purple curve uses the DCUNet backbone, while the orange curve uses NCSN++.

你的橙色PESQ感觉是正常的，DCUnet我没有实验

**Comment by dorispei** [comment]
> > > Why is my training so much worse than yours?
> > 
> > 
> > ![图片](https://private-user-images.githubusercontent.com/44772675/391777561-fd89e76c-61b4-4137-bf6c-f716cbe15f0d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyMTM0OTIsIm5iZiI6MTczMzIxMzE5MiwicGF0aCI6Ii80NDc3MjY3NS8zOTE3Nzc1NjEtZmQ4OWU3NmMtNjFiNC00MTM3LWJmNmMtZjcxNmNiZTE1ZjBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAzVDA4MDYzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjMmU4YmNlZTZjMDBiYmJjNjg5NTIxOWRkYWU5MTZkZTA2MGYzNDkwNjMzN2YzZTU5ZjEzOGRiOTY3MGU1NGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FglgQsMPEMWV4a4pSnKm2bivxhbdia7GPFWuwFHTloI) I have the same problem. Have you resolved it? Below is the log of my training. The dataset used is VoiceBank-Demand. The purple curve uses the DCUNet backbone, while the orange curve uses NCSN++.
> 
> 你的橙色PESQ感觉是正常的，DCUnet我没有实验
It seems alright. The orange PESQ is less than 2.2, while the paper reports 2.48±0.58 for VB-DMD. Could it be because mine is validset, not testset? Also, training with VB-DMD using 58spks should yield better results; after about 23 epochs, PESQ = 2.38.

**Comment by libai2345** [comment]
I noticed that the pesq results of your first few training sessions started from 1.0. Why did the pesq results after that start directly from >2? Did you modify anything?

**Comment by a897456** [comment]
> I noticed that the pesq results of your first few training sessions started from 1.0. Why did the pesq results after that start directly from >2? Did you modify anything?

我没关注PESQ起点大小，但是我刚查看了多条训练曲线，PESQ的起点确实存在差异，当然这些曲线的配置确实有修改。但最终目的是PESQ落点。因为起点的高低，好像并未影响到落点。

**Comment by libai2345** [comment]
> > 我注意到你前几次训练的 pesq 结果从 1.0 开始。为什么之后的 pesq 结果直接从 >2 开始？你修改了什么吗？
> 
> 我没有关注PESQ起点大小，但是我刚查看了多条训练曲线，PESQ的起点确实存在差异，当然这些曲线的配置确实有修改。但最终目的是PESQ落点。因为起点的高低，需要产生影响到达落点。
Excuse me, could you tell me in detail what configurations you have modified? My training results are not ideal, the pesq is only 1.5.

**Comment by a897456** [comment]
> > > 我注意到你前几次训练的 pesq 结果从 1.0 开始。为什么之后的 pesq 结果直接从 >2 开始？你修改了什么吗？
> > 
> > 
> > 我没有关注PESQ起点大小，但是我刚查看了多条训练曲线，PESQ的起点确实存在差异，当然这些曲线的配置确实有修改。但最终目的是PESQ落点。因为起点的高低，需要产生影响到达落点。
> > Excuse me, could you tell me in detail what configurations you have modified? My training results are not ideal, the pesq is only 1.5.

我这边把网络改了，配置参数你可能不适用。所以我看看你的配置  看看能不能看出问题。

**Comment by libai2345** [comment]
> > > > 我注意到你前几次训练的 pesq 结果从 1.0 开始。为什么之后的 pesq 结果直接从 >2 开始？你修改了什么吗？
> > > 
> > > 
> > > 我没有关注PESQ起点大小，但是我刚查看了多条训练曲线，PESQ的起点确实存在差异，当然这些曲线的配置确实有修改。但最终目的是PESQ落点。因为起点的高低，需要产生影响到达点落点。
> > > 请问您具体修改了哪些配置？我的训练结果并不理想，pesq只有1.5。
> 
> 我把网络改了，参数配置你可能不适用。所以我看看你的看看能不能看出问题。

My configuration parameters have not changed, which is surprising! I don't know why the training results are so poor.
